.github/workflows/reusable-azure-ai-translate.yml

```diff
@@ -93,42 +91,52 @@ jobs:
         id: datetime
         run: echo "timestamp=$(date -u '+%B %d, %Y at %H:%M UTC')" >> "$GITHUB_OUTPUT"
 
-      - name: Create Pull Request to update translations
+      - name: Configure Git identity
         if: steps.check_pr.outputs.found == 'false'
-        env:
-          GH_TOKEN: ${{ secrets.NEWFOLD_ACCESS_TOKEN }}
-          BRANCH_NAME: ${{ steps.vars.outputs.BRANCH }}
-          TIMESTAMP: ${{ steps.datetime.outputs.timestamp }}
         run: |
-          # Configure Git identity
           git config --global user.name "github-actions"
           git config --global user.email "github-actions@github.com"
 
-          # Ensure remote uses token for Git push
-          git remote set-url origin "https://x-access-token:${GH_TOKEN}@github.com/${{ github.repository }}.git"
-
-          # Create or reset to the branch
+      - name: Create or reset to branch
+        if: steps.check_pr.outputs.found == 'false'
+        env:
+          BRANCH_NAME: ${{ steps.vars.outputs.BRANCH }}
```

Comment:
Just catching this, but can we use a different name for this instead of `vars` to avoid confusion with the `vars` [context in actions for repository/organization variables](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/store-information-in-variables)?